etabotsite/etabotapp/urls.py

```diff
@@ -35,6 +36,7 @@
     url(r'^api/parse_projects/', ParseTMSprojects.as_view(), name="estimate_tms"),
     url(r'^api/atlassian_oauth', AtlassianOAuth.as_view(), name='atlassian_oauth'),
     url(r'^api/vote/', VoteView.as_view(), name="vote"),
```

Comment:
@ZachElkins , could api/vote/ be replaced with api/user_communication/?
If so please do. Also pleas add unit test for the api/user_communication/ in test_views_models.py following existing unit tests as examples